doc/experimenter.md

```diff
@@ -103,10 +103,10 @@ A likert multi stimulus page creates a multi-stimulus likert rating.
 * **type** must be likert_multi_stimulus.
 * **id** Identifier of the page.
 * **name** Name of the page (is shown as title)
+* **content** Content (HTML) of the page. The content is shown on the upper part of the page.
 * **mustRate** If set to true, the participant must rate all stimuli.
-* **mustPlayback** If set to true, the participant must fully play back all stimuli. 
-* **reference** Filepath to the reference stimulus (WAV file).
-* **stimuli** A map of stimuli representing three conditions. The key is the name of the condition. The value is the filepath to the stimulus (WAV file).
+* **mustPlayback** If set to ended, the participant must fully play back all stimuli to the end. If set to processUpdate, the participant must start playing back all stimuli before rating becomes possible.
```

Comment:
ended -> `ended`

Reply:
processUpdate -> `processUpdate`

Reply:
name of processUpdate is not clear to me what I would expect.

Reply:
I agree with processUpdate being unclear, but that happens to be the name of the event that's emitted whenever audio plays back. In order to keep the implementation light-weight, I just used the existing internal event name. So either that event name needs to be changed or a convoluted "translation" layer needs to be added.